WcaOnRails/spec/features/claim_wca_id_spec.rb

```diff
@@ -87,7 +87,9 @@
       fill_in "Birthdate", with: "1988-02-03"
       click_button "Claim WCA ID"
 
-      expect(page.find(".alert.alert-danger")).to have_content("There have been too many incorrect claims for this WCA ID. Please contact the WCA Results Team to resolve this.")
+      # Based on WRT request here: https://github.com/thewca/worldcubeassociation.org/pull/3666#discussion_r253315031,
+      # the expected message for this is users.errors.wca_id_no_birthdate_html
+      expect(page.find(".alert.alert-danger")).to have_content("We do not have a birthdate recorded for this WCA ID. Please contact the WCA Results Team")
```

Comment:
Not a huge deal, but this feels like weird text to show people... it's not that we don't have a birthdate recorded, it's that the birthdate is probably wrong. I think a message like this would make more sense:

> There have been too many incorrect claims for this WCA ID, so we've locked this id to prevent future attempts to claim it. It's possible we have the wrong date of birth on file. Please contact the WCA Results Team using <a href=\"%{dob_form_path}\">this dedicated form</a>.

Reply:
I'd definitely agree with that, though you actually said it sounded good to you [there](https://github.com/thewca/worldcubeassociation.org/pull/3666#discussion_r253337865) :p

Basically I'd use whatever the WRT thinks it's best here.

Reply:
Haha, yeah, I didn't think through this when I wrote that comment :cry: